cms/jinja2/templates/base.html

```diff
@@ -80,6 +80,9 @@
 {# fmt:on #}
 {% block head %}
     <link rel="stylesheet" href="{{ static ('css/main.css') }}">
+    {% if extra_media %}
```

Comment:
Could not find references for this. One from the original implementation?

Reply:
Thanks. Yes: this was part of the original implementation, but I also didn't know whether it was used elsewhere, as it seemed a fairly generic pattern. I'll remove it.